src/coreclr/nativeaot/Runtime/thread.cpp

```diff
@@ -1232,7 +1232,10 @@ void Thread::EnsureRuntimeInitialized()
     if (g_RuntimeInitializationCallback != NULL)
     {
         if (g_RuntimeInitializationCallback() != 0)
+        {
+            PalPrintFatalError("\nFatal error. Runtime failed to initialize.\n");
```

Comment:
```suggestion
            PalPrintFatalError("\nFatal error. .NET runtime failed to initialize.\n");
```
Runtime is a bit ambiguous.